### ISSUE TITLE
Provide versioned PURLs for wikidata

### ISSUE THREAD

**Issue by cmungall** [DiseaseOntology/HumanDiseaseOntology#192]
https://www.wikidata.org/wiki/Q206901

under the OMIM xrefs, I see:

```
archive URL
https://raw.githubusercontent.com/DiseaseOntology/HumanDiseaseOntology/686b3fcb0d50fb5e5c34d9088cdc0b7dd7bb6771/src/ontology/doid.owl
```

this is not great as a versioned file. What if DO moves away from GitHub?

OBO provides guidelines about versioned PURLs, these should follow a certain URI structure. It's actually very easy to manage these using GitHub, but at the same time be independent of GitHub.

You can see an example of how this could work in the [PATO editors README](https://github.com/pato-ontology/pato/blob/master/src/ontology/README-editors.md)

Basically, the idea is
- remove the existing `releases` folder from github
- use the GitHub release mechanism to tag releases (very easy)
- Registere a release purl with OBO that redirects to the the versioned raw file in GitHub

If you go this route I can provide help with the setup

cc @andrawaag

**Comment by stuppie** [comment]
Thanks for implementing this! I'll use the reference URL in this format for wikidata:
http://purl.obolibrary.org/obo/doid/releases/2017-01-27/doid.owl

**Comment by cmungall** [comment]
I would strongly recommend wikidata using the OBO PURLs.

The DOID release managers are responsible for making sure these redirect by editing this file:

https://github.com/OBOFoundry/purl.obolibrary.org/blob/master/config/doid.yml

Some groups add an entry for every release. Other groups make a single redirect PURL (we do this for all the ontologies I am responsible for).

It's up to DOID what the purl redirects to: it could be S3, a /releases/ subfolder in master of this repo... however, based on extensive experience with managing ontologies I **strongly** recommend doing this via the GitHub release mechanism as we do for PATO, ENVO etc, see instructions above. In general, whether it's for software or ontologies __checking releases into the master branch is a bad idea that won't scale and you will regret later__

off soapbox...

**Comment by stuppie** [comment]
Ok, for the next run, I'll update the references to "stated in" [Disease Ontology release 2017-01-27](https://www.wikidata.org/wiki/Q28556593), which has the redirect PURL as the archive URL.

**Comment by stuppie** [comment]
For the OBOFoundry yaml file, it looks like all that needs to be done is add the following?

```
- prefix: /releases/
  replacement: https://github.com/DiseaseOntology/HumanDiseaseOntology/blob/master/src/ontology/releases/
  tests:
  - from: /releases/2017-01-27/doid.owl
    to: https://github.com/DiseaseOntology/HumanDiseaseOntology/blob/master/src/ontology/releases/2017-01-27/doid.owl
```

**Comment by cmungall** [comment]
almost - you need the raw URL

https://raw.githubusercontent.com/DiseaseOntology/HumanDiseaseOntology/master/src/ontology/releases/

If you make the PR, @elviram or someone from DOID +1s it, an OBO person will merge

**Comment by elviram** [comment]
@stuppie I thought we had this discussion with GeneWiki group some time ago. And it was decided that just stating the retrieval date from the DO was enough and that a link to the file at that point in time was not needed.

**Comment by stuppie** [comment]
@elviram Ideally we point to the specific release with a PURL for that release. From my understanding, using github as the domain was temporary until it was set up in OBOFoundry? Alternatively, we can also just use the retrieval date and http://ontologies.berkeleybop.org/doid.owl if needed..

**Comment by elviram** [comment]
Resolved in hipchat.
